example-app/SantokuApp/src/features/qa-question/components/CommentsWithCount.tsx

```diff
@@ -0,0 +1,18 @@
+import {Box, Text} from 'bases/ui/common';
+import {QuestionAnswerIllustration} from 'bases/ui/illustration/QuestionAnswerIllustration';
+import React from 'react';
+
+type AnswerWithCountProps = {
+  count: number;
+};
+export const AnswerWithCount: React.FC<AnswerWithCountProps> = ({count}) => {
```

Comment:
Isn't it CommentsWithCount

Reply:
79723dc2a9d7436ffe1c2eccbd2f59fe87261bfd